source/llassetgen-rendering/main.cpp

```diff
@@ -307,6 +322,7 @@ class Window : public WindowQt {
     int samplerIndex = 0;
     glm::mat4 transform3D = glm::mat4();
     glm::mat4 projection = glm::perspective(45.f, 1.f, 0.0001f, 100.f);
+    uint superSampling = 0;
```

Comment:
I would either use a C++ unsigned int type if that works, or use `gl::uint` to make clear where `uint` is coming from and why it is used.

Reply:
uint is from Qt. I use the c++ unsigend int type now, if that helps for readability.